src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -235,6 +244,7 @@ class ActivityFeed extends React.Component<Props, State> {
         const { isInputOpen } = this.state;
         const hasAnnotationCreatePermission = getProp(file, ['permissions', PERMISSION_CAN_CREATE_ANNOTATIONS], false);
         const hasCommentPermission = getProp(file, 'permissions.can_comment', false);
+        const hasResolvePermission = getProp(file, ['permissions', PERMISSION_CAN_RESOLVE], false);
```

Comment:
Should we have a test for this logic?

Reply:
I've removed this as it actually was a wrong approach. The `can_resolve` permission is on a comment (item) level, not file, so it should only be checked in the `Comment` itself (similar to how it's done with `can_edit` or `can_delete` permissions). This will be implemented in a future PR.